List-Comprehensions.ipynb

```diff
@@ -95,8 +95,13 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    l_say("Calender. wow. even less impressed. Just import calendar and....")
</pre><p>Typo in first Calend<strong>a</strong>r</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>